cf-overrides.yml

```diff
@@ -0,0 +1,6 @@
+BaseImageTag: "2.3-v1.0.2"
+PreBundle: |-
+  RUN apt-get update && apt-get install -y dnsutils netcat
+  COPY Rakefile /app/Rakefile
+  COPY github_bitbucket_deployer.gemspec /app/github_bitbucket_deployer.gemspec
```

Comment:
Copying the gemspec is part of the standard template now, so you can remove this line.